adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -119,6 +119,8 @@ func (b *BM25Searcher) wand(
 		models.PropertyTokenizationLowercase,
 		models.PropertyTokenizationWhitespace,
 		models.PropertyTokenizationField,
+		models.PropertyTokenizationTrigram,
+		models.PropertyTokenizationGse,
 	}
```

Comment:
can we have a list of all tokenizers in the Tokenizer package and use that everywhere? Then it is easier to to not forget adding a new one

Reply:
Sounds like a great idea, where should we put the list?

Reply:
I think the adapters/repos/db/helpers/tokenizer.go file is a natural place for this and you already defining the list there anyways